models/Recipe.model.js

```diff
@@ -1,10 +1,24 @@
+//1. IMPORTACIONES
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
+
+//2. Schema
 const recipeSchema = new Schema({
```

Comment:
Buen trabajo Rod note que dentro del modelo hicieron falta algunas especificaciones para cada propiedad, dentro de esta sugerencia hay un enum: [ ...], este indica que dentro de esta propiedad solo las caracteristicas de la lista son permitidas, tambien agregue una imagen por defecto como aparece en la parte de abajo 😉
```suggestion
const recipeSchema = new Schema({
    title: {type: String, required: true, unique: true},
    level: {type: String, enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]},
    ingredients: {type: Array},
    cuisine: {type: String, required: true},
    dishType: {type: String, enum: ["breakfast", "main_course", "soup", "snack", "drink", "dessert", "other"]},
    image: {type: String, default: "https://images.media-allrecipes.com/images/75131.jpg"},
    duration: {type: Number, min: 0},
    creator: {type: String},
    created: {type: Date, default: Date.now }
});```